packages/account/src/Sections/Security/ClosingAccount/closing-account-reason-form.tsx

```diff
@@ -0,0 +1,253 @@
+                                onChangeCheckbox(values, field.name, setFieldValue);
+                            }}
+                        />
+                    )}
+                </Field>
+                <Field name='difficult-transactions'>
+                    {({ field }: FormikValues) => (
+                        <Checkbox
+                            disabled={is_checkbox_disabled && !values[field.name as keyof TFormValues]}
+                            className='closing-account-reasons__checkbox'
+                            {...field}
+                            label={localize('Making deposits and withdrawals is difficult.')}
+                            onChange={() => {
+                                onChangeCheckbox(values, field.name, setFieldValue);
+                            }}
+                        />
+                    )}
+                </Field>
+                <Field name='lack-of-features'>
+                    {({ field }: FormikValues) => (
+                        <Checkbox
+                            disabled={is_checkbox_disabled && !values[field.name as keyof TFormValues]}
+                            className='closing-account-reasons__checkbox'
+                            {...field}
+                            label={localize('The platforms lack key features or functionality.')}
+                            onChange={() => {
+                                onChangeCheckbox(values, field.name, setFieldValue);
+                            }}
+                        />
+                    )}
+                </Field>
+                <Field name='unsatisfactory-service'>
+                    {({ field }: FormikValues) => (
+                        <Checkbox
+                            disabled={is_checkbox_disabled && !values[field.name as keyof TFormValues]}
+                            className='closing-account-reasons__checkbox'
+                            {...field}
+                            label={localize('Customer service was unsatisfactory.')}
+                            onChange={() => {
+                                onChangeCheckbox(values, field.name, setFieldValue);
+                            }}
+                        />
+                    )}
+                </Field>
+                <Field name='other-reasons'>
+                    {({ field }: FormikValues) => (
+                        <Checkbox
+                            disabled={is_checkbox_disabled && !values[field.name as keyof TFormValues]}
+                            className='closing-account-reasons__checkbox'
+                            {...field}
+                            label={localize('I’m closing my account for other reasons.')}
+                            onChange={() => {
+                                onChangeCheckbox(values, field.name, setFieldValue);
+                            }}
+                        />
+                    )}
+                </Field>
```

Comment:
As these checkboxes are all same in structure, and the only props which are changing are `name` & `label`
We can make an array of objects like

`
    const ACCOUNT_CLOSING_REASON_FIELDS = [{
        name: 'unsatisfactory-service',
        label: localize('Customer service was unsatisfactory')
    }, {
        name: 'other-reasons',
        label: localize('I'm closing my account for other reasons')
    }]
`

And we don't need to handle checkbox separately, Formik can handle them by itself
Ref: https://formik.org/docs/examples/checkboxes